src/main/java/org/sagebionetworks/web/client/widget/entity/tabs/AbstractTablesTab.java

```diff
@@ -60,13 +60,14 @@ public abstract class AbstractTablesTab implements TablesTabView.Presenter, Quer
 	public static final String TABLE_QUERY_PREFIX = "query/";
 
 	private static final String VERSION_ALERT_DRAFT_DATASET_TITLE = "This is a Draft Version of the Dataset";
-	private static final String VERSION_ALERT_DRAFT_DATASET_MESSAGE = "Administrators and Editors can edit this version and create a Stable Version for distribution. Go to the Version History to view the Stable Versions.";
+	private static final String VERSION_ALERT_DRAFT_DATASET_MESSAGE = "Administrators can edit this version and create a Stable Version for distribution. Go to the Version History to view the Stable Versions";
 
 	private static final String VERSION_ALERT_OLD_SNAPSHOT_DATASET_TITLE = "There is a newer Stable Version of this Dataset";
 	private static final String VERSION_ALERT_OLD_SNAPSHOT_DATASET_MESSAGE = "Go to the latest Stable Version, or view the Version History for all versions.";
 	public static final String GO_TO_LATEST_STABLE_VERSION = "Go to Latest Stable Version";
 	public static final String NO_STABLE_VERSIONS_OF_THIS_DATASET = "There are currently no Stable Versions of this Dataset";
 
+	private static final String DATASET_CREATED_BY_HELP_TEXT = "This is the user who created this Dataset. This may not be the same person who generated the files in this Dataset, or who originally uploaded these files to Synapse.";
```

Comment:
Looks like this branch was not created from the develop branch (somehow commits from SWC-6011 made it in here)

Reply:
Forgot to branch off of develop, should be fixed now that I merged develop into this branch